roles/confluent.zookeeper/tasks/main.yml

```diff
@@ -146,3 +148,23 @@
   when:
     - health_checks_enabled|bool
     - not ansible_check_mode
+
+- name: "Set zookeeper ACLs: {{zookeeper_sasl_digestmd5_superuser_login}} - Full, anyone - Read"
```

Comment:
This task runs on zookeeper[0] host, but uses the kafka_broker jaas file. Should it be using the zookeeper jaas file?
Are you testing on where zookeeper and kafka are on the same host?

Reply:
Good point. Yes, i had them on the same hosts.

I think It won't be enought to just replace it with zookeeper_jaas.conf, it has to contain client section first.
May be it's even worth placing it on separate file, to not mislead anyone 

```
Client {
    org.apache.zookeeper.server.auth.DigestLoginModule required
    username="{{zookeeper_sasl_digestmd5_superuser_login}}"
    password="{{zookeeper_sasl_digestmd5_superuser_password}}";
};
```